tests/test_ignite.py

```diff
@@ -307,12 +304,20 @@ def test_ignite_dataset_with_plain_client_with_interleave(self):
 
     """
     self._clear_env()
+
+    igds_local = ignite_io.IgniteDataset(
+        cache_name="SQL_PUBLIC_TEST_CACHE",
+        schema_host="localhost",
+        host='localhost',
+        port=10800)
+
+    # TODO: this is a workaround due to failure to build a TypeSpec for
```

Comment:
@yongtang I think this should have technically worked before, but the failure seems due to being in non-eager mode. After the 0.8.0 release, we will probably want to change these over to eager mode, so this is just a temporary fix. WDYT?